health-services/project-factory/src/server/validators/microplanValidators.ts

```
@@ -0,0 +1,300 @@
+        }
+    }
+}
+
+export function validateTargetsForMicroplanCampaigns(data: any, errors: any, localizedTargetColumnNames: any, localizationMap?: { [key: string]: string }) {
+    for (const key in data) {
+        if (key !== getLocalizedName(getBoundaryTabName(), localizationMap) && key !== getLocalizedName(config?.values?.readMeTab, localizationMap)) {
+            if (Array.isArray(data[key])) {
+                const boundaryData = data[key];
+                boundaryData.forEach((obj: any, index: number) => {
+                    var totalTarget = 0, totalTargetSumFromColumns = 0;
+                    for (let i = 0; i < localizedTargetColumnNames.length; i++) {
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)